virt-v2v/pkg/customize/scripts/rhel/run/network_config_util.sh

```diff
@@ -133,9 +169,19 @@ udev_from_netplan() {
         return 0
     fi
 
+    # Function to check if netplan supports the 'get' subcommand
+    netplan_supports_get() {
+        netplan get 2>/dev/null
+        return $?
+    }
+
     # netplan with root dir
     netplan_get() {
-        netplan get --root-dir "$NETPLAN_DIR" "$@" 2>/dev/null
+        if netplan_supports_get; then
+            netplan get --root-dir "$NETPLAN_DIR" "$@" 2>/dev/null
```

Comment:
Could we pass the stderr to the `log`?

Reply:
now redirect to file descriptor 3 ( our log file ), and then to screen

Reply:
Yeah sounds good to me